staging/src/k8s.io/kube-aggregator/pkg/client/informers/internalversion/apiregistration/internalversion/apiservice.go

```diff
@@ -51,23 +52,31 @@ func NewAPIServiceInformer(client internalclientset.Interface, resyncPeriod time
 	return NewFilteredAPIServiceInformer(client, resyncPeriod, indexers, nil)
 }
 
+func NewAPIServiceInformerWithMultiTenancy(client internalclientset.Interface, resyncPeriod time.Duration, indexers cache.Indexers, tenant string) cache.SharedIndexInformer {
+	return NewFilteredAPIServiceInformerWithMultiTenancy(client, resyncPeriod, indexers, nil, tenant)
+}
+
 // NewFilteredAPIServiceInformer constructs a new informer for APIService type.
 // Always prefer using an informer factory to get a shared informer instead of getting an independent
 // one. This reduces memory footprint and number of connections to the server.
 func NewFilteredAPIServiceInformer(client internalclientset.Interface, resyncPeriod time.Duration, indexers cache.Indexers, tweakListOptions internalinterfaces.TweakListOptionsFunc) cache.SharedIndexInformer {
+	return NewFilteredAPIServiceInformerWithMultiTenancy(client, resyncPeriod, indexers, tweakListOptions, "system")
```

Comment:
what is "system"?

Reply:
also consider add a short comment about why a const tenant is used here to make sure it's on purpose.

Reply:
This is a file auto-generated by codegen. Codegen is not smart enough to import the dependency needed for a constant variable (it is good when importing dependencies for  types/packages/funcs). So I don't have a way to include the dependency of package "metav1" in the file generated.